config/testdata/conf.good.yml

```diff
@@ -109,7 +109,8 @@ scrape_configs:
 - job_name: service-kubernetes
 
   kubernetes_sd_configs:
-  - api_servers:
+  - kind: service_endpoints
```

Comment:
> Service endpoints are a rare configuration

AFAIK they are probably the most widely used configuration, albeit they have the problems we've discussed meaning we want to steer people away from it to pod discovery, but with enhanced labelling.